File: java/pe006.java
//file: pe006.java

public class pe006 {
  public static void main( String[] args ) {
    int sumsq = 0;
    for (int i = 0; i < 100; ++i) {
      sumsq += (i+1) * (i+1);
    }

    int sqsum = (101 * 50) * (101 * 50);
    int result = sqsum - sumsq;

    System.out.println(result);
  }
}

File: go/pe006.go
package main

import (
	"fmt"
)

func main() {
	var sumsq int
	sumsq = 0
	for i := 0; i < 100; i++ {
		sumsq += (i + 1) * (i + 1)
	}
	sqsum := (101 * 50) * (101 * 50)
	fmt.Printf("%d\n", sqsum-sumsq)
}
